app/controllers/calls_controller.rb

```diff
@@ -2,32 +2,39 @@ class CallsController < ApplicationController
 
   after_filter :set_header
 
+
+
   # Public: initiates the call process via a request from twillio
   #
   # CallSid - default param from twilio (required)
   def start
     response = Twilio::TwiML::Response.new do |r|
-      r.Say 'We need you to connect with your congressperson and senator. We are going to put you in touch with '
+      r.Play AudioFileFetcher.file_for_key('intro_message')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.